src/containers/main/ReferContainer/index.tsx

```diff
@@ -226,6 +226,60 @@ export const ReferContainer: React.FC<IProps> = () => {
 						/>
 					</div>
 				)}
+
+				<div className={styles.how_it_works_container}>
+					<div className={styles.how_it_works}>How it works?</div>
+					<div className={styles.how_it_works_section}>
+						<div>
+							<div className={styles.how_it_works_steps}>Step 1: Choose a Affiliate ID</div>
+							<p className={styles.how_it_works_content}>
+								Go to the REX Referral page:{' '}
+								<a
```

Comment:
![image](https://user-images.githubusercontent.com/69639595/220081932-fe1d3479-ad57-4aa7-ba22-716186181e0d.png)

The URL has changed and needs to be updated to:
https://ricochet-exchange.eth.limo/#/refer

Furthermore, please pull latest changes to V2, I merged one of your PRs and don't want a merge conflict.